src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -46,9 +46,10 @@ export class TransformMarkdownAction extends Action {
 
 export class MarkdownTextTransformer {
 
-	private _notebookEditor: INotebookEditor;
```

Comment:
Instead of making this public I think it'd be better to add it as an optional argument to the constructor